doc/rst/developer/bestPractices/ContributorInfo.rst

```diff
@@ -134,16 +148,81 @@ pass testing for ``start_test -performance``).
 
 .. _Creating a Simple Test: https://github.com/chapel-lang/chapel/blob/master/doc/rst/developer/bestPractices/TestSystem.rst#creating-a-simple-test
 
+.. _Design
+
+Design
+------
+
+.. _Creating a design issue
+
+Creating a design issue
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The design issue should aim to clearly and concisely present some design
+decisions. Here are some strategies to make a good design issue:
+
+- Summarize issue at the top, preferably include concise code example(s).
+- Try not to make initial issue too long in order to lower the barrier for
+  someone getting involved.
+- Clearly define the problem this proposal aims to solve.
+- It can be useful to consider alternative solutions to the problem and
+  describe the trade-offs among them.
+- A design issue should have two developers not involved in the effort to review.
+- To find reviewers, send an email to chapel-developers_.
```

Comment:
Maybe we can add that drawing parallels from common programming languages is a plus.

Reply:
I see that this is mentioned below. But I don't see any harm in adding examples from other languages from the get-go rather than sprinkling them in the discussion. So, still think that we can add a bullet for that here.

Reply:
@mppf  and I chatted about this and concluded it's not always the responsibility of the person opening the design issue to do this, hence why we put it below. I am not opposed to mentioning or referencing that section up here though.

Reply:
I am fine with leaving this part as-is.